google/cloud/forseti/common/opencensus/tracing.py

```diff
@@ -0,0 +1,108 @@
+# Copyright 2017 The Forseti Security Authors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""Forseti OpenCensus gRPC tracing setup."""
+
+from opencensus.trace import config_integration
+from opencensus.trace import execution_context
+from opencensus.trace.exporters import file_exporter
+from opencensus.trace.exporters import stackdriver_exporter
+from opencensus.trace.exporters.transports import background_thread
+from opencensus.trace.ext.grpc import client_interceptor
+from opencensus.trace.ext.grpc import server_interceptor
+from opencensus.trace.samplers import always_on
+from opencensus.trace.tracer import Tracer
+
+from google.cloud.forseti.common.util import logger
+
+LOGGER = logger.get_logger(__name__)
+DEFAULT_INTEGRATIONS = ['requests', 'sqlalchemy']
+
+
+def create_client_interceptor(endpoint):
+    """Create gRPC client interceptor.
+
+    Args:
+        endpoint (str): The gRPC channel endpoint (e.g: localhost:5001).
+
+    Returns:
+        OpenCensusClientInterceptor: a gRPC client-side interceptor.
+    """
+    exporter = create_exporter()
+    tracer = Tracer(exporter=exporter)
+    return client_interceptor.OpenCensusClientInterceptor(
+        tracer,
+        host_port=endpoint)
+
+
+def create_server_interceptor(extras=True):
+    """Create gRPC server interceptor.
+
+    Args:
+        extras (bool): If set to True, also trace integration libraries.
+
+    Returns:
+        OpenCensusServerInterceptor: a gRPC server-side interceptor.
+    """
+
+    exporter = create_exporter()
+    sampler = always_on.AlwaysOnSampler()
+    if extras:
+        trace_integrations()
```

Comment:
Would it be very hard to assert on the state of `config_integration` after `config_integration.trace_integrations()` is called?  I would think that it should be possible.

If that is too hard, can you please put a comment on the return statement in that the returned values are only used for testing?  So that in the future we can remember the return is there for testing purposes.